packages/pharos/src/components/image-card/pharos-image-card.ts

```diff
@@ -254,7 +254,9 @@ export class PharosImageCard extends ScopedRegistryMixin(FocusMixin(PharosElemen
   private _renderHoverMetadata(): TemplateResult | typeof nothing {
     return this.subtle
       ? html`<div class="card__metadata--hover">
-          <strong class="card__title--hover">${this.title}</strong><slot name="metadata"></slot>
+          ${this._renderCheckbox()}
+          <strong class="card__title--hover">${this.title}</strong>
+          <slot name="metadata"></slot>
```

Comment:
Do we need to hide the metadata when the card is selected?

Reply:
AFAIK yes, although I haven't had a chance to run this by Lori yet to ensure its looking as she expected

Reply:
Did give Lori a moment to look a this and you are right. We do now hide the metadata on selection

https://user-images.githubusercontent.com/13315416/191797133-d84c0f24-8ac7-49ee-9918-c0669d15dbff.mov